### ISSUE TITLE
Regarding the XC3 bypass

### ISSUE THREAD

**Issue by zeffy** [Xyene/BNSBoost#7]
Hey @Xyene long time! :)

Just wanted to add here, since you've implemented a XIGNCODE bypass you should probably remove or modify the phrasing on this line in the README: 

> BNSBoost never touches the game client (Client.exe), only its launcher.

Also it would probably be good to give attribution to who originally made the bypass. Dunno where you got this `x3.xem` from since it appears to be (slightly) modified, but the code originally came from [VirtualPuppet/**XignCode3-bypass**].

[VirtualPuppet/**XignCode3-bypass**]: https://github.com/VirtualPuppet/XignCode3-bypass

**Comment by Xyene** [comment]
Hey! :)

Those are good points; I was going to redo the readme sometime today. Booted up BNS for the first time in months last night and a friend requested that functionality, so I just hacked something up as quick as possible.

I don't feel great about shipping a prebuilt binary I pulled off Reddit -- even if people seemed to trust it -- so I'll probably be including the original repo as a subproject and building it alongside everything else... Once I catch a bit more free time.
